app/components/op_primer/expandable_list_component.rb

```diff
@@ -25,33 +27,29 @@
 #
 # See COPYRIGHT and LICENSE files for more details.
 #++
+#
+module OpPrimer
+  class ExpandableListComponent < Primer::Component
+    include ApplicationHelper
+    include OpTurbo::Streamable
+    include OpPrimer::ComponentHelpers
 
-require "spec_helper"
+    CUTOFF_LIMIT = 10
```

Comment:
The new cutoff limit is defined but I believe in all of the calls in the productive code, it is still overwritten with 5 which contradicts the specification in the work package.